wagtail/documents/templates/wagtaildocs/documents/list.html

```diff
@@ -8,6 +8,7 @@
     <col width="16%" />
     <thead>
         <tr class="table-headers">
+            {% include 'wagtailadmin/bulk_actions/select_all_checkbox_cell.html' %}
```

Comment:
These cells are wider than they should be for me.
![image](https://user-images.githubusercontent.com/13856515/128313039-452a70c7-7b84-40b1-898f-331388b6da13.png)

I would expect them to be similarly sized as in the page explorer.